### PR TITLE
Admin Tool: Yank Rubygems

### PR DESCRIPTION
**Pull request by arunagw**
We currently have a script to yank Rubygems for a user.
This PR makes it possible to do it via Admin UI so that
we don't have to do it via a script.

* Added an avo action to yank all rubygems for a user.
* Used this new action to Rubygem admin show page.

This is how it looks in the UI

Monosnap handle9 — RubyGems org (development) 2023-04-04 10 pm-46-22

### CODE REVIEW

File: app/avo/actions/yank_rubygems_for_user.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "Are you sure you would like to yank all rubygems for user #{record.handle} with #{record.email}?"
+  }
+
+  self.confirm_button_label = "Yank all Rubygems"
+
+  class ActionHandler < ActionHandler
+    def handle_model(user)
+      rubygems = user.rubygems
+
+      User.transaction do
```
Comment (segiddins) [inline app/avo/actions/yank_rubygems_for_user.rb:19]: handle_model is already called within a transaction so I don't think we need the nested transaction

File: app/avo/actions/yank_rubygems_for_user.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        rubygems.each do |rubygem|
```
Comment (segiddins) [inline app/avo/actions/yank_rubygems_for_user.rb:20]: Since this is a relation, should we use find_each ?

**Review by segiddins — COMMENTED** [review]
Looks good with a test!

File: app/services/yank_rubygem_service.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class YankRubygemService
```
Comment (simi) [inline app/services/yank_rubygem_service.rb:1]: Is it worth it to introduce "service" folder (and pattern) for this small usecase? Wouldn't be enough to hide the logic in `User` model for example?

**Review by simi — APPROVED** [review]

**Pull request merged**